scripts/lib/payload.js

```diff
@@ -135,8 +181,18 @@ window.Payload.LFI = {
 }
 
 window.Payload.SSTI = {
-  flaskRCE: value => {
+  Jinja2:{
```

Comment:
A style issue

Space is needed after colon.

Reply:
Fixed now.